### DISCUSSION TITLE
How to dynamically expand (increase) and reduce distributed nodes

### DISCUSSION THREAD

**Discussion by hktalent** [etcd-io/etcd#14843] [Q&A] [answered]
How to dynamically expand (increase) and reduce distributed nodes

thanks

**Comment by hktalent** [comment]
I don't know whether there is a web http api or other apis that provide calls

**Comment by lavacat** [comment, marked as answer]
Have you seen `Runtime reconfiguration` [guide](https://etcd.io/docs/v3.5/op-guide/runtime-configuration/)?
